napari_clusters_plotter/_Qt_code.py

```diff
@@ -354,6 +354,41 @@ def algorithm_choice(name: str, value, options: dict, label: str):
     return container, choice_list
 
 
+def create_options_dropdown(name: str, value, options: dict, label: str):
```

Comment:
I renamed this function to make it more general and replaced its references. I am keeping the `algorithm_choice` for backwards compatibility, although I doubt someone is actually using it directly.